ros2_batch_job/__init__.py

```diff
@@ -1 +0,0 @@
-DEFAULT_REPOS_URL = 'https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos'
```

Comment:
This was the only content in this file, but I'm not familiar enough with python convention to know if I should :bomb: the file entirely or leave it empty.